f2/src/ConfirmationSummary.js

```diff
@@ -36,6 +36,7 @@ export const ConfirmationSummary = () => {
     ...testdata.formData.anonymous,
     ...data.formData.anonymous,
   }
+  const { fyiForm } = data.formData
```

Comment:
This PR appears to integrate changes that already exist in master such as FYI Flow. What happened?

Reply:
I just did rebase yesterday, my change is just 1 file

Reply:
Do not know why these pop up